can t connect using rmapi getting: panic:runtime error

I need help here: getting following error trying to connect to rnfakecloud. With UI i can connect without problem:

Enter one-time code (go to https://my.remarkable.com/connect/desktop): pgvhqxwe
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x908ef0]

goroutine 1 [running]:
github.com/juruen/rmapi/transport.HttpClientCtx.addAuthorization(0xc000199770, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/rmapi/rmapi/transport/transport.go:62 +0x50
github.com/juruen/rmapi/transport.HttpClientCtx.Request(0xc000199770, 0x0, 0x0, 0x0, 0x0, 0x0, 0xa41c60, 0x4, 0xc0002d1fb0, 0x2b, ...)
	/home/runner/work/rmapi/rmapi/transport/transport.go:168 +0x10f
github.com/juruen/rmapi/transport.HttpClientCtx.httpRawReq(0xc000199770, 0x0, 0x0, 0x0, 0x0, 0x0, 0xa41c60, 0x4, 0xc0002d1fb0, 0x2b, ...)
	/home/runner/work/rmapi/rmapi/transport/transport.go:130 +0x198
github.com/juruen/rmapi/transport.HttpClientCtx.Post(...)
	/home/runner/work/rmapi/rmapi/transport/transport.go:98
github.com/juruen/rmapi/api.newDeviceToken(0xc000199740, 0xc0002dd1e3, 0x8, 0x0, 0x0, 0x0, 0xc0001996e0)
	/home/runner/work/rmapi/rmapi/api/auth.go:90 +0x233
github.com/juruen/rmapi/api.AuthHttpCtx(0xc000070000, 0xc0000201d0)
	/home/runner/work/rmapi/rmapi/api/auth.go:29 +0x1e9
main.main()
	/home/runner/work/rmapi/rmapi/main.go:32 +0x193

--- a/transport/transport.go
+++ b/transport/transport.go
@@ -163,7 +163,10 @@ func (ctx HttpClientCtx) httpRawReq(authType AuthType, verb, url string, reqBody
 }
 
 func (ctx HttpClientCtx) Request(authType AuthType, verb, url string, body io.Reader) (*http.Response, error) {
-	request, _ := http.NewRequest(verb, url, body)
+	request, err := http.NewRequest(verb, url, body)
+	if err != nil {
+		return nil, err
+	}
 
 	ctx.addAuthorization(request, authType)
 	request.Header.Add("User-Agent", RmapiUserAGent)

